drivers/clk/ralink/Kconfig

```diff
@@ -9,3 +9,10 @@ config CLK_MT7621
 	select MFD_SYSCON
 	help
 	  This driver supports MediaTek MT7621 basic clocks.
+
+config CLK_MTMIPS
+	bool "Clock driver for MTMIPS SoCS"
```

Comment:
There's still an uppercase S on SoCs. After that, everything looks good to me to be submitted.

Reply:
Before submission I am wondering what should be do with 'arch/mips/boot/dts/ralink' dtsi files. All of these are a bit out of date and are minimal and I don't think anybody is using them. I guess I must update all of them also in this patchset to align things?